setup.py

```diff
@@ -15,10 +15,6 @@
         "turnkeyml.tools",
         "turnkeyml.tools.discovery",
         "turnkeyml.sequence",
-        "turnkeyml.run",
-        "turnkeyml.run.onnxrt",
-        "turnkeyml.run.tensorrt",
-        "turnkeyml.run.torchrt",
```

Comment:
Can we also get rid of any of the install requirements (move those requirements to the plugins)?

Reply:
I don't know - that would be a separate project.